src/Microsoft.Identity.Client/ApiConfig/AcquireTokenSilentParameterBuilder.cs

```diff
@@ -80,26 +89,6 @@ public AcquireTokenSilentParameterBuilder WithForceRefresh(bool forceRefresh)
             return this;
         }
 
-        /// <summary>
-        /// </summary>
-        /// <param name="loginHint"></param>
-        /// <returns></returns>
-        public AcquireTokenSilentParameterBuilder WithLoginHint(string loginHint)
```

Comment:
I removed these 2 methods from the public API because account or login hint are mandatory. So they are part of the ctor.

Reply:
Actually I was also thinking that we could not mandate those if there is 1 and only one account in the accounts (just use it). (if there are more than one we throw)
This was requested by customers ...

Reply:
Yes, this is implemented.